### DISCUSSION TITLE
Prefix not working anymore

### DISCUSSION THREAD

**Discussion by Mijawel** [openai/whisper#818] [Q&A]
Anyone noticed the prefix parameter not working anymore (in DecodingOptions)?

I'm not 100% sure when it stopped working but my last update of whisper model was on Oct 19 and since I re-pulled the repo and model on Jan 4, every time I enter some text in the prefix parameter I get an empty string as output.

**Comment by zw123han** [comment]
https://github.com/openai/whisper/pull/1358 Does this fix your issue? A token was missing.

**Reply by Mijawel** [reply]
Thanks, not sure yet as have found a workaround since then but appreciate the fix!
